p2p/p2p_test.go

```diff
@@ -14,11 +14,24 @@ import (
 	"github.com/bitmark-inc/logger"
 )
 
+// Note: test will fail very badly if something is using this port
+//       lots of "bind: address already in use" will be produced
+const unusedPortForTesting = 32136
```

Comment:
to avoid test on same port, I would try to use random port for test, e.g.:

```
port := random.Intn(30000) + 30000  // guarantees the port is in range 30000 ~ 60000
```

Reply:
random may produce the same pseudo random sequence, so I used crypto/rand to be sure it will be truly random.  Also changed range to IANA range for dynamic ports